### ISSUE TITLE
cryptocat extension for firefox, chrome:// url prefix

### ISSUE THREAD

**Issue by mykhael** [cryptocat/cryptocat#705]
This is just a question so this could be labeled as `question` if needed, but out of curiosity why does the Cryptocat extension for Firefox display the `chrome://` prefix in the address bar if the current browser isn't chrome?

**Comment by geekman-rohit** [comment]
Chrome refers to all the things inside the browses interface.. Google just named it browser Chrome, for reasons I do not know, perhaps they couldn't find a new name. All browsers including chrome itself use that prefix to refer to things inside it, like extension in this case.

**Comment by mykhael** [comment]
@geekman-rohit Yes, I completely understand your explanation however couldn't the Firefox extension based `resources://` prefix have been used instead of `chrome://`, still confused why this is used in the Firefox extension when it's not Google Chrome at all.

**Comment by Q---** [comment]
Way too many (otherwise intelligent) people see "chrome" in the address bar and instantly think it's associated with Google Chrome.  The URL identifier 'chrome' existed long before Google Chrome did.

I've even heard people complain that cryptocat is not secure, because it uses "the google chrome application interface"
